File: 上下轮播图/上下轮播图/ViewController.m
//
//  ViewController.m
//  上下轮播图
//
//  Created by 周冰烽 on 2017/10/30.
//  Copyright © 2017年 周冰烽. All rights reserved.
//

#import "ViewController.h"
#import "loopView.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    loopView *loop = [[loopView alloc]init];
    loop.frame = CGRectMake(100, 200, 400, 250);
    [self.view addSubview:loop];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: 自定义cell编辑状态/自定义cell编辑状态/ViewController.swift
//
//  ViewController.swift
//  自定义cell编辑状态
//
//  Created by 周冰烽 on 2017/12/12.
//  Copyright © 2017年 周冰烽. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let tableView = UITableView(frame: CGRect(x: 0, y: 64, width: kScreenWidth, height: kScreenHeight - 64), style: .plain)
        view.addSubview(tableView)
        tableView.snp.makeConstraints { (make) in
            
        }
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

